website/docs/docs/cloud/git/setup-azure.md

```diff
@@ -1,37 +1,37 @@
 ---
 title: "Set up Azure DevOps"
 id: "setup-azure"
-description: "You can set up your Azure DevOps by creating an Azure AD app and adding it to dbt Cloud."
+description: "You can set up your Azure DevOps by creating a Microsoft Entra ID app and adding it to dbt Cloud."
 sidebar_label: "Set up Azure DevOps"
 ---
 
 <Snippet path="available-enterprise-tier-only" />
 
 ## Overview
 
-To use our native integration with Azure DevOps in dbt Cloud, an account admin needs to set up an Azure Active Directory (Azure AD) app. We recommend setting up a separate [Azure AD application than used for SSO](/docs/cloud/manage-access/set-up-sso-azure-active-directory).
+To use our native integration with Azure DevOps in dbt Cloud, an account admin needs to set up a Microsoft Entra ID app. We recommend setting up a separate [Microsoft Entra ID application than used for SSO](/docs/cloud/manage-access/set-up-sso-microsoft-entra-id).
 
-1. [Register an Azure AD app](#register-an-azure-ad-app).
+1. [Register a Microsoft Entra ID app](#register-a-microsoft-entra-id-app).
 2. [Add permissions to your new app](#add-permissions-to-your-new-app).
 3. [Add another redirect URI](#add-another-redirect-uri).
 4. [Connect Azure DevOps to your new app](#connect-azure-devops-to-your-new-app).
-5. [Add your Azure AD app to dbt Cloud](#add-your-azure-ad-app-to-dbt-cloud).
+5. [Add your Microsoft Entra ID app to dbt Cloud](#add-your-microsoft-entra-id-app-to-dbt-cloud).
 
-Once the Azure AD app is added to dbt Cloud, an account admin must also connect a service user via OAuth, which will be used to power headless actions in dbt Cloud such as deployment runs and CI.
+Once the Microsoft Entra ID app is added to dbt Cloud, an account admin must also connect a service user via OAuth, which will be used to power headless actions in dbt Cloud such as deployment runs and CI.
 1. [Connecting a Service User](#connecting-a-service-user).
```

Comment:
fixing link
```suggestion
1. [Connecting a Service User](#connect-a-service-user).
```

Reply:
hey @nghi-ly , i'll reopen this again and commit there sugg as it's still not going to work since the h2 is 'connect'

Reply:
weird, i remember clicking "Commit suggestion" for this. doing it again...

Reply:
i went to update and it's already committed 🤷‍♀️ resolving ... again

![Screenshot 2024-03-21 at 9 10 31 AM](https://github.com/dbt-labs/docs.getdbt.com/assets/107218380/3a7fc38d-7b88-4992-a3f6-1c740bdd132b)